packages/react/src/primitives/StepperField/StepperField.tsx

```diff
@@ -103,7 +105,7 @@ const StepperFieldPrimitive: Primitive<StepperFieldProps, 'input'> = (
             onClick={handleDecrease}
             size={size}
           >
-            <IconRemove data-testid={DECREASE_ICON} size={size} />
```

Comment:
Should we remove the size prop from line 40 as well?

Reply:
Good catch! Will do